sys/include/net/netopt.h

```diff
@@ -532,6 +532,8 @@ typedef enum {
      */
     NETOPT_TX_RETRIES_NEEDED,
 
+    NETOPT_BLE_CTX,             /**< set radio context (channel, CRC, AA) */
```

Comment:
Arghs, this doesn't uphold the new doc style for `netopt`s introduced in #8655. Fix provided in #8892.